Package.resolved

```diff
@@ -86,8 +86,8 @@
         "package": "SwiftSyntax",
         "repositoryURL": "https://github.com/apple/swift-syntax.git",
         "state": {
-          "branch": "0.40200.0",
-          "revision": "f20c0bcbfb4b07e107c9851350217cb2217f0cb4",
+          "branch": "swift-5.3-DEVELOPMENT-SNAPSHOT-2020-09-07-a",
```

Comment:
Just to confirm, is that the version you want to link?

Reply:
Swift 5.3 is still in development.

Reply:
yeah, this is for the next release :P